src/python/WMCore/WMSpec/WMTask.py

```diff
@@ -1616,6 +1616,22 @@ def getAcquisitionEra(self):
         """
         return getattr(self.data.parameters, 'acquisitionEra', None)
 
+    def setCampaignName(self, campaign):
+        """
```

Comment:
Please update the docstring according to the guidelines: https://github.com/dmwm/WMCore/blob/master/CONTRIBUTING.rst#project-docstrings-best-practices